### PR TITLE
Separate VS Code Extension Prerequisites page

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###
Related with https://github.com/Samsung/tizen-docs/pull/743,
it is required to separate the prerequisites part from VS Code Extension for .NET page.

### CODE REVIEW

File: docs/application/vscode-ext/dotnet.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# Visual Studio Code Extension for Tizen .NET
+
+Visual Studio Code Extension for Tizen (VS Code Extension for Tizen) is a Visual Studio Code extension that enables you to develop Tizen .NET applications easily using Visual Studio Code. This topic covers 2 different ways to perform the installation: installing from the Marketplace or installing from a VSIX file.
+
+Before installing VS Code Extension for Tizen, check [Prerequisites of Visual Studio Code Extension for Tizen](index.md).
+
+## Installing from the Marketplace
```
Comment (singa2000) [inline docs/application/vscode-ext/dotnet.md]: Install from the Marketplace ?

Comment (mijin85cho) [reply docs/application/vscode-ext/dotnet.md]: updated

File: docs/application/vscode-ext/index.md [hunk 2]
```diff
@@ -14,11 +9,12 @@ To work with Visual Studio Code Extension for Tizen, your computer must have:
   - Windows&reg; 7/10 (64 bit)
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/vscode-ext/index.md]: @singa2000 Is this update also valid to Web?

[the remark this answers is not in the document]
Comment (singa2000) [reply docs/application/vscode-ext/index.md]: Web is also no problem.
It's valid to Web.

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/vscode-ext/index.md]: Updated.

**Comment by mijin85cho** [comment]
Thanks for the review

**Pull request merged**